src/TextArea/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import asInput, { inputProps } from '../asInput';
+
+function Text(props) {
+  return (
+    <textarea
+      id={props.id}
+      className={classNames(props.className)}
+      type="text"
+      name={props.name}
+      value={props.value}
+      placeholder={props.placeholder}
+      aria-describedby={props.describedBy}
+      onChange={props.onChange}
+      onBlur={props.onBlur}
+      aria-invalid={!props.isValid}
+      disabled={props.disabled}
+      required={props.required}
+    />
+  );
+}
+
+Text.propTypes = inputProps;
```

Comment:
Were you thinking about adding `propTypes` for props here that aren't covered by `inputProps` (for example, `isValid`)?

I think you could do something like

```javascript
Text.propTypes = Object.assign({}, inputProps, {
  id: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.bool]),
  placeholder: PropTypes.string,
  describedBy: PropTypes.string,
  isValid: PropTypes.bool,
  ...
});
```
but there could be a better way

Reply:
I kind of have the same answer for all of your feedback haha -- I used the InputText component as the base for the creation of this component, so the syntax choices essentially match up to that. If we think these changes should be made I'm happy to make them, but if not I'm also happy to have the two Components share similarities for uniformity sake.